test/functional/mempool_accept.py

```diff
@@ -104,7 +104,7 @@ def run_test(self):
         ))['hex']
         tx.deserialize(BytesIO(hex_str_to_bytes(raw_tx_final)))
         self.check_mempool_result(
-            result_expected=[{'txid': tx.rehash(), 'allowed': True}],
+            result_expected=[{'txid': tx.rehash(), 'allowed': True, 'fee': (int(coin['amount']) - (0.025)) * COIN}],
```

Comment:
please no floating point in tests. Decimal is good enough

Reply:
I am a little confused here, i used `int` because `Decimal` wasn't asserting with `int` type return value in fee. `0.025` is the output value in btc which is already used few lines above while constructing the tx. What exactly you are suggesting to change here?

Reply:
Nevemind. I wrote this test and used floating point in the first place. :sweat_smile: :cold_sweat: